src/commands/api/uploadAppSettingsFromApi.ts

```diff
@@ -6,8 +6,8 @@ import { callWithTelemetryAndErrorHandling, IActionContext } from "vscode-azuree
 import { IAppSettingsClient } from "../../vscode-azurefunctions.api";
 import { uploadAppSettingsInternal } from "../appSettings/uploadAppSettings";
 
-export async function uploadAppSettingsFromApi(client: IAppSettingsClient): Promise<void> {
+export async function uploadAppSettingsFromApi(client: IAppSettingsClient, exclude?: (RegExp | string)[]): Promise<void> {
     return await callWithTelemetryAndErrorHandling('api.uploadAppSettings', async (context: IActionContext) => {
-        await uploadAppSettingsInternal(context, client);
+        await uploadAppSettingsInternal(context, client, undefined, exclude);
```

Comment:
One thing I like to do is put the called function variable name next to `undefined` when it is passed, so it's understandable what _would_ go there if it wasn't `undefined` without needing to look at the called function.
i.e.
```suggestion
        await uploadAppSettingsInternal(context, client, /* workspaceFolder */ undefined, exclude);
```

Reply:
Good suggestion. 

Also, not sure if you know about this but VS Code recently added a setting to provide inlay hints for parameter names:

![image](https://user-images.githubusercontent.com/12476526/149037773-59699405-65b8-40c7-aeb8-62de0307196a.png)

![image](https://user-images.githubusercontent.com/12476526/149037846-b3338eb2-9a90-454a-baba-8b7c1f8cfd31.png)

Reply:
I did not know about this! That is awesome and I'm turning it on now.